backends-velox/src/test/scala/org/apache/gluten/execution/ScalarFunctionsValidateSuite.scala

```diff
@@ -918,6 +918,24 @@ abstract class ScalarFunctionsValidateSuite extends FunctionsValidateTest {
     }
   }
 
+  testWithSpecifiedSparkVersion("mask", Some("3.4")) {
```

Comment:
Could you explain a bit why we need this specific version of Spark?

Reply:
hey @rui-mo , mask only avaliable for 3.4+ spark. this for only run it for 3.4+ version

Reply:
In this case, do we need to add version 3.5 also?

Reply:
@rui-mo, this specifies the version range. For this case, it will be run for spark >=3.4.

Reply:
this is specific the min version is 3.4, so 3.5 also be covered.

Reply:
Thanks for clarifying.